.github/workflows/docker-image.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches: [ "master" ]
   pull_request:
     branches: [ "master" ]
+concurrency:
+  group: ${{ format('docker-image-{0}', github.event.pull_request.number || github.sha) }}
```

Comment:
```suggestion
  group: ${{ format('docker-ci-{0}', github.event.pull_request.number || github.sha) }}
```

Reply:
The `name` as defined on line 1 and the key of the job (`build` in this case). Job also has an optional property `name`.